packages/widget-chat/src/components/avatar/styles.css

```diff
@@ -1,13 +1,18 @@
+.avatar {
+  display: flex;
+  flex: 1 1 auto;
+}
+
 .avatar-letter {
   background-color: darkgray;
   width: 40px;
   height: 40px;
   border-radius: 20px;
-  margin: 5px;
+  margin: auto;
 }
 
 .avatar-letter strong {
```

Comment:
Is avatar-letter ever not going to be strong? If it's the same style then maybe just remove the strong tag and combine the the `.avatar-letter` and the `.avatar-letter strong` styles into one.